### ISSUE TITLE
Sudden increase in loss!

### ISSUE THREAD

**Issue by kannadaraj** [NVIDIA/tacotron2#358]
Hi,

  I am training model on a new dataset with default hparams and the database is around 20 hours of curated data.. Training is good until a point and for unknown reason, the loss increases drastically as shown in the figure below. the alignment was being learnt properly. But it never recovers after this sudden increase in loss. What might be causing this kind of loss increase. Please can someone explain. Thanks.

![sudden_loss_increase](https://user-images.githubusercontent.com/17844576/82925955-54053f80-9f33-11ea-99d0-10c7eb1ded3d.PNG)

**Comment by ksaidin** [comment]
Have you tried to anneal learning rate and resume training from just before loss jump?

**Comment by kannadaraj** [comment]
@ksaidin thanks for the reply.  I havent tried annealing yet. But training on LJ speech doesn't require any annealing fro good traininig. Can annealing solve this issue?

Also, i started from before the loss jump. It takes very long time like almost 60K more iterations to get to the original loss value. That is also so unusual.

I will try annealing. Do you have any other suggestions..

**Comment by ksaidin** [comment]
@CookiePPP is suggesting to anneal to somewhere around 1e-4 depending on your batch size. [>>344](https://github.com/NVIDIA/tacotron2/issues/344#issuecomment-626066212)

You can find @rafaelvalle suggesting to anneal learning rate multiple times throughout issues of this repo.

**Comment by kannadaraj** [comment]
@ksaidin thanks a lot..
